lib/SPIRVProducerPass.cpp

```diff
@@ -1923,6 +1923,7 @@ spv::BuiltIn SPIRVProducerPass::GetBuiltin(StringRef Name) const {
       .Case("__spirv_WorkgroupSize", spv::BuiltInWorkgroupSize)
       .Case("__spirv_NumWorkgroups", spv::BuiltInNumWorkgroups)
       .Case("__spirv_WorkgroupId", spv::BuiltInWorkgroupId)
+      .Case("__spirv_WorkDim", spv::BuiltInWorkDim)
```

Comment:
This is a bit dodgy/confusing IMHO as we're not using the SPIR-V `WorkDim` built-in. Pragmatically speaking, no harm done though.